WooCommerce/Classes/ViewRelated/Dashboard/Settings/Settings/SettingsViewModel.swift

```diff
@@ -280,11 +280,17 @@ private extension SettingsViewModel {
         let storeSettingsSection: Section? = {
             var rows: [Row] = [.storeName]
 
-            let site = stores.sessionManager.defaultSite
-            if site?.isJetpackCPConnected == true ||
-                (site?.isNonJetpackSite == true &&
-                 featureFlagService.isFeatureFlagEnabled(.jetpackSetupWithApplicationPassword)) {
-                rows.append(.installJetpack)
+            if let site = stores.sessionManager.defaultSite {
+                if site.isJetpackCPConnected == true ||
+                    (site.isNonJetpackSite == true &&
+                     featureFlagService.isFeatureFlagEnabled(.jetpackSetupWithApplicationPassword)) {
+                    rows.append(.installJetpack)
+                }
+
+                let themesUseCase = ThemeEligibilityUseCase(featureFlagService: featureFlagService)
```

Comment:
Nit: We can also keep the initializer empty as we default to the default feature flag service. Only explicit injection is needed for tests.

```suggestion
                let themesUseCase = ThemeEligibilityUseCase()
```